common/lib/client/auth.js

```diff
@@ -239,8 +239,15 @@ var Auth = (function() {
 				}
 			}
 			tokenRequestCallback = function(params, cb) {
-				var authHeaders = Utils.mixin({accept: 'application/json'}, authOptions.authHeaders);
-				Http.getUri(rest, authOptions.authUrl, authHeaders || {}, Utils.mixin(params, authOptions.authParams), function(err, body, headers, unpacked) {
+				var authHeaders = Utils.mixin({accept: 'application/json'}, authOptions.authHeaders),
+						authParams = Utils.mixin(params, authOptions.authParams);
+				Logger.logAction(Logger.LOG_MICRO, 'Auth.requestToken().tokenRequestCallback', 'Sending; ' + authOptions.authUrl + '; Params: ' + JSON.stringify(authParams));
+				Http.getUri(rest, authOptions.authUrl, authHeaders || {}, authParams, function(err, body, headers, unpacked) {
+					if (err) {
+						Logger.logAction(Logger.LOG_MICRO, 'Auth.requestToken().tokenRequestCallback', 'Received Error; ' + JSON.stringify(err));
+					} else {
+						Logger.logAction(Logger.LOG_MICRO, 'Auth.requestToken().tokenRequestCallback', 'Received; body: ' + JSON.stringify(BufferUtils.isBuffer(body) ? body.toString() : body));
```

Comment:
Because I am not sure what object the `body` will aways be, so I felt it's safer to do this as opposed to ending up with `[Object object]` which is all too common when not using stringify.  What is the downside?